model_info/viwiki.articletopic.md

```diff
@@ -816,146 +816,146 @@ Model Information:
 		Geography.Regions.Asia.East Asia             0.946
-		Geography.Regions.Asia.North Asia            0.968
-		Geography.Regions.Asia.South Asia            0.932
-		Geography.Regions.Asia.Southeast Asia        0.908
-		Geography.Regions.Asia.West Asia             0.95
-		Geography.Regions.Europe.Eastern Europe      0.951
-		Geography.Regions.Europe.Europe*             0.926
-		Geography.Regions.Europe.Northern Europe     0.921
+		Geography.Regions.Asia.North Asia            0.935
+		Geography.Regions.Asia.South Asia            0.929
+		Geography.Regions.Asia.Southeast Asia        0.909
+		Geography.Regions.Asia.West Asia             0.946
+		Geography.Regions.Europe.Eastern Europe      0.932
+		Geography.Regions.Europe.Europe*             0.922
+		Geography.Regions.Europe.Northern Europe     0.915
 		Geography.Regions.Europe.Southern Europe     0.934
-		Geography.Regions.Europe.Western Europe      0.94
-		Geography.Regions.Oceania                    0.923
-		History and Society.Business and economics   0.97
-		History and Society.Education                0.972
+		Geography.Regions.Europe.Western Europe      0.945
+		Geography.Regions.Oceania                    0.912
+		History and Society.Business and economics   0.968
+		History and Society.Education                0.973
 		History and Society.History                  0.948
-		History and Society.Military and warfare     0.975
-		History and Society.Politics and government  0.922
-		History and Society.Society                  0.899
-		History and Society.Transportation           0.986
-		STEM.Biology                                 0.986
+		History and Society.Military and warfare     0.974
+		History and Society.Politics and government  0.957
+		History and Society.Society                  0.905
+		History and Society.Transportation           0.988
+		STEM.Biology                                 0.985
 		STEM.Chemistry                               0.988
-		STEM.Computing                               0.986
+		STEM.Computing                               0.987
 		STEM.Earth and environment                   0.974
-		STEM.Engineering                             0.983
-		STEM.Libraries & Information                 0.964
-		STEM.Mathematics                             0.984
+		STEM.Engineering                             0.985
+		STEM.Libraries & Information                 0.968
+		STEM.Mathematics                             0.987
 		STEM.Medicine & Health                       0.976
 		STEM.Physics                                 0.987
-		STEM.STEM*                                   0.982
-		STEM.Space                                   0.99
-		STEM.Technology                              0.977
+		STEM.STEM*                                   0.98
+		STEM.Space                                   0.989
+		STEM.Technology                              0.975
 		-------------------------------------------  -----
-	pr_auc (micro=0.706, macro=0.541):
+	pr_auc (micro=0.703, macro=0.534):
```

Comment:
This makes it clear that it's the new vectors that are undercutting our fitness -- not something else in the pipeline.  I'm working on rebuilding the viwiki vectors to make sure nothing went strangely there.  I'll have a fresh 10k vector set to try some time today (July 13th)

Reply:
Improvements :-D 

pr_auc (micro=0.82, macro=0.672)